README.md

```diff
@@ -43,13 +43,8 @@ public class SimpleStrangeDemo {
 
     public static void main(String[] args) {
         Program p = new Program(2);
-        Step s = new Step();
```

Comment:
For a demo, I'd like to keep everything explainable step by step, with each step on 1 line.
For internal/test code, it can be optimized like you did in the other patches.

Reply:
Done